<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>QgsPatternDialogBase</class>
 <widget class="QDialog" name="QgsPatternDialogBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Select a fill pattern</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>10</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" colspan="4" >
    <widget class="Q3ButtonGroup" name="buttonGroup1" >
     <property name="title" >
      <string/>
     </property>
     <property name="exclusive" >
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>10</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="0" >
       <widget class="QPushButton" name="solid" >
        <property name="text" >
         <string/>
        </property>
        <property name="checkable" >
         <bool>true</bool>
        </property>
        <property name="default" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QPushButton" name="fdiag" >
        <property name="text" >
         <string/>
        </property>
        <property name="checkable" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QPushButton" name="dense4" >
        <property name="text" >
         <string/>
        </property>
        <property name="checkable" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QPushButton" name="horizontal" >
        <property name="text" >
         <string/>
        </property>
        <property name="checkable" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QPushButton" name="dense5" >
        <property name="text" >
         <string/>
        </property>
        <property name="checkable" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QPushButton" name="diagcross" >
        <property name="text" >
         <string/>
        </property>
        <property name="checkable" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QPushButton" name="dense1" >
        <property name="text" >
         <string/>
        </property>
        <property name="checkable" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="2" >
       <widget class="QPushButton" name="dense6" >
        <property name="text" >
         <string/>
        </property>
        <property name="checkable" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QPushButton" name="vertical" >
        <property name="text" >
         <string/>
        </property>
        <property name="checkable" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QPushButton" name="cross" >
        <property name="text" >
         <string/>
        </property>
        <property name="checkable" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QPushButton" name="dense2" >
        <property name="text" >
         <string/>
        </property>
        <property name="checkable" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0" >
       <widget class="QPushButton" name="bdiag" >
        <property name="text" >
         <string/>
        </property>
        <property name="checkable" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1" >
       <widget class="QPushButton" name="dense3" >
        <property name="text" >
         <string/>
        </property>
        <property name="checkable" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="2" >
       <widget class="QPushButton" name="dense7" >
        <property name="text" >
         <string/>
        </property>
        <property name="checkable" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="2" >
       <widget class="QPushButton" name="nopen" >
        <property name="toolTip" >
         <string>No Fill</string>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="checkable" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="2" >
    <widget class="QPushButton" name="cancelbutton" >
     <property name="text" >
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QPushButton" name="okbutton" >
     <property name="text" >
      <string>Ok</string>
     </property>
     <property name="default" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="3" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>41</width>
       <height>21</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>41</width>
       <height>21</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3ButtonGroup</class>
   <extends></extends>
   <header>Qt3Support/Q3ButtonGroup</header>
   <container>1</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>solid</tabstop>
  <tabstop>fdiag</tabstop>
  <tabstop>dense4</tabstop>
  <tabstop>horizontal</tabstop>
  <tabstop>diagcross</tabstop>
  <tabstop>dense5</tabstop>
  <tabstop>vertical</tabstop>
  <tabstop>dense1</tabstop>
  <tabstop>dense6</tabstop>
  <tabstop>cross</tabstop>
  <tabstop>dense2</tabstop>
  <tabstop>dense7</tabstop>
  <tabstop>bdiag</tabstop>
  <tabstop>dense3</tabstop>
  <tabstop>okbutton</tabstop>
  <tabstop>cancelbutton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
